Concentration/ViewController.swift

```diff
@@ -41,18 +65,37 @@ class ViewController: UIViewController
                 button.backgroundColor = UIColor.white
             } else {
                 button.setTitle("", for: UIControl.State.normal)
-                button.backgroundColor = card.isMatched ? UIColor.clear : UIColor.systemOrange
+                button.backgroundColor = card.isMatched ? UIColor.clear : self.theme.cardBackColor
             }
         }
+        self.flipCountLabel.text = "Flips: \(self.game.flipCount)"
+        self.scoreLabel.text = "Score: \(self.game.score)"
     }
 
     private func emoji(for card: Card) -> String {
-        if emoji[card.identifier] == nil, Self.emojiChoices.count > 0 {
-            let randIndex = Int.random(in: 0..<Self.emojiChoices.count)
-            emoji[card.identifier] = Self.emojiChoices.remove(at: randIndex)
+        if emoji[card.identifier] == nil, self.theme.emojis.count > 0 {
+            let randIndex = Int.random(in: 0..<self.theme.emojis.count)
+            emoji[card.identifier] = self.theme.emojis.remove(at: randIndex)
         }
 
         return emoji[card.identifier] ?? "?"
     }
+
 }
 
+extension ViewController {
+
+    private static var themes: Array<Theme> = [
```

Comment:
can we make it part of `Theme` type? And make a `getRandom()` static func there?